draft-irtf-cfrg-opaque.md

```diff
@@ -1052,7 +1052,34 @@ for binding between the underlying OPRF protocol messages and the KE session.
 Next, we present three instantiations of OPAQUE - with HMQV, 3DH and SIGMA-I.
 {{I-D.sullivan-tls-opaque}} discusses integration with TLS 1.3 {{RFC8446}}.
 
-## Instantiation of OPAQUE with HMQV and 3DH {#SecHmqv}
+## Key Schedule Utility Functions
+
+The key derivation procedures for HMQV, 3DH, and SIGMA-I instantiations
+all make use of the functions below, re-purposed from TLS 1.3 {{?RFC8446}}.
+
+~~~
+HKDF-Expand-Label(Secret, Label, Context, Length) =
+  HKDF-Expand(Secret, HkdfLabel, Length)
+~~~
+
+Where HkdfLabel is specified as:
+
+~~~
+struct {
+   uint16 length = Length;
+   opaque label<7..255> = "tls13 " + Label;
```

Comment:
"OPAQUE" instead of "tls13" I guess (use upper case OPAQUE to distinguish from the opaque type.

Reply:
Copy-paste bug :) Will fix!